afinn/data/AFINN-no-165.txt

```diff
@@ -0,0 +1,3214 @@
+forskrekket	-2
+innlagt på sykehus	-2
+fiendtlig	-2
+høker	-2
+klem	2
+enorm	1
+klemmer	2
+human	2
+ydmyk	1
+ydmyket	-3
+ydmykelse	-3
+humoristisk	2
+humor	2
+sult	-2
+hurra	5
+sårer	-2
+gjør vondt	-2
+hyklersk	-2
+hysteri	-3
+hysterisk	-3
+idioti	-3
+idiotisk	-2
+uvitenhet	-2
+overse	-1
+ignorert	-2
+ignorerer	-1
+jeg vil	-2
+skjebnesvangre	-2
+ulovlig	-3
+uekte	-3
+analfabetisme	-2
+ulogisk	-2
+fantasifull	2
+immobilisert	-1
+udødelig	2
+immun	1
+svekke	-2
+hemmet	-2
+svekkelse	-2
+svekker	-2
+utålmodig	-2
+riksrett	-3
+hindret	-2
+vanskeliggjør	-2
+hemmende	-2
+ufullkommen	-2
+betydning	1
+viktig	2
+pålegge	-1
+pålagt	-1
+pålegger	-1
+imponere	3
+imponert	3
+imponerer	3
+imponerende	3
+upassende	-2
+forbedre	2
+forbedret	2
+forbedring	2
+forbedrer	2
+manglende evne	-2
+passivitet	-2
+utilstrekkelig	-2
+uforvarende	-2
+ikke i stand til	-2
+uføre	-2
+røkelse	-2
+ansikte	-2
+incen	-2
+inkompetanse	-2
+inkompetent	-2
+ufullstendig	-1
+ubegripelig	-2
+hensynsløs	-2
+upraktisk	-2
+økt	1
+ubesluttsom	-2
+uforgjengelig	2
+likegyldighet	-2
+likegyldig	-2
+indignert	-2
+indignasjon	-2
+indoktrinere	-2
+indoktrinert	-2
+indoktrinerende	-2
+ubønnhørlige	-3
+utilgivelig	-3
+ineffektiv	-2
+ineffektivt	-2
+ørkesløshet	-2
+ineffektivitet	-2
+udugelig	-2
+betatt	2
+forelskelse	2
+infisere	-2
+infisert	-2
+infeksjon	-2
+infeksjoner	-2
+smittsomme	-2
+infiserer	-2
+mindreverdig	-2
+søke	-2
+betent	-2
+påføre	-2
+påført	-2
+påfører	-2
+innflytelsesrik	2
+overtredelse	-2
+gjøre rasende	-2
+hemme	-1
+urettferdighet	-2
+innovere	1
+innovasjon	1
+nyskapende	2
+virkningsløs	-2
+inkvisisjonen	-2
+nysgjerrige	2
+sinnsyk	-2
+galskap	-3
+utrygg	-2
+ufølsom	-2
+ufølsomhet	-2
+ubetydelig	-2
+smakløs	-2
+søvnløshet	-2
+inspirasjon	2
+inspirerende	3
+inspirere	2
+inspirert	2
+inspirerer	2
+insuffisiens	-2
+fornærmelse	-2
+fornærmet	-2
+fornærmelser	-2
+intakt	2
+integritet	2
+intens	1
+renter	1
+interessert	2
+interessant	2
+interesser	1
+avhørt	-1
+avbryte	-2
+fortrolighet	2
+skremme	-2
+skremmer	-2
+trusler	-2
+uforsonlighet	-2
+innviklet	2
+intriger	1
+invasjon	-1
+uovervinnelig	2
+invitere	1
+innbydende	1
+usårbar	2
+ironisk	-1
+ironi	-1
+irrasjonell	-1
+uopprettelig	-2
+uforklarlige	-2
+uimotståelig	2
+uansvarlig	-2
+irreversibelt	-1
+isolert	-1
+kløe	-2
+kvikk	2
+sjalu	-2
+sjalusi	-2
+dust	-3
+juvel	1
+juveler	1
+spøkefull	2
+bli med	1
+vits	2
+vitser	2
+gledelig	3
+gledesløs	-2
+skvetten	-1
+rette	2
+berettiget	2
+tilbakeslag	-3
+kidnappe	-2
+kidnappet	-2
+kidnappinger	-2
+drepe	-3
+drept	-3
+dreper	-3
+snill	2
+på en måte	0
+snillere	2
+kysse	2
+kudos	3
+mangel på	-2
+etterslep	-2
+landemerke	2
+bortfalt	-1
+latter	3
+lo	1
+ler	1
+laugh	1
+lanserte	1
+rettssak	-2
+søksmål	-2
+lat	-1
+ledelse	1
+ledende	2
+lekke	-1
+lekket	-1
+permisjon	-1
+lovlig	1
+skånsom	1
+dødelighet	-2
+sløv	-2
+løgner	-3
+løgnere	-3
+injurier	-2
+løy	-2
+livløs	-1
+livsredder	4
+letthjertet	1
+som	2
+likandes	2
+likte	2
+liker	2
+smak	2
+begrensning	-2
+grenser	-1
+prosedyre, rettstvist	-1
```

Comment:
There is a comma here. maybe you should erase "prosedyre" and just maintain "rettstvist" on -1

Reply:
Thank you for your quick response! I will go through these reviews and with your reccomendation submit a new PR.

Yes, you are right, this should just be "rettstvist".